DWAPlannerROS:

# Robot Configuration Parameters
  max_vel_x: 0.8
  min_vel_x: -0.6

  max_vel_y: 0.0
  min_vel_y: 0.0

# The velocity when robot is moving in a straight line
  max_vel_trans:  0.8
  min_vel_trans:  0.1

  max_vel_theta: 5
  min_vel_theta: 1.37

  acc_lim_x: 2.5
  acc_lim_y: 0.0
  acc_lim_theta: 3.2

# Goal Tolerance Parametes
  xy_goal_tolerance: 0.35
  yaw_goal_tolerance: 0.15
  latch_xy_goal_tolerance: false

# Forward Simulation Parameters
  sim_time: 1.5
  vx_samples: 20
  vy_samples: 0
  vth_samples: 40

# Trajectory Scoring Parameters
  path_distance_bias: 32
  goal_distance_bias: 25
  occdist_scale: 0.2
  forward_point_distance: 0.4
  stop_time_buffer: 0.5
  scaling_speed: 0.25
  max_scaling_factor: 0.2

# Oscillation Prevention Parameters
  oscillation_reset_dist: 0.1

# Debugging
  # publish_traj_pc : true
  # publish_cost_grid_pc: true